### PR TITLE
Adding Rock Paper Scissors console game with score tracking

### PR DESCRIPTION
**Pull request by sahnavin123**
# Description:
The project involves creating a console-based game of Rock Paper Scissors to be played against the computer. Functions will be used to generate random selections for the computer and to determine the winner of each round. A separate function will be used to play a 5 round game and keep score.

## Features:
- Blank HTML document with a script tag added
- Function called getComputerChoice() to randomly return 'Rock', 'Paper', or 'Scissors'
- Function called playRound() to determine the winner of a every round
- Function called game to play a 5 round game and keep score
- Use of prompt() to get input from the user

## Manual Testing:
- Verified that the getComputerChoice() function returned 'Rock', 'Paper', or 'Scissors' randomly
- Tested the playRound() function by passing different player and computer selections and checking the returned winner string
- Player choice was obtained through prompt
- Tested the game function by playing a 5 round game and checking the score and winner reported at the end

## project video

https://user-images.githubusercontent.com/95286412/235128961-9737dfec-b1f8-4007-ad1b-b35a672fe3bc.mp4

### CODE REVIEW

**Comment by devanuj97** [comment]
@sahnavin123 Can you printout the computer choices as well in the console this would be more helpful to identify & verify the result

File: script.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            switch (computerChoice) {
+                case 'rock':
+                    return ("computer");
+                case 'paper':
+                    return ("you");
+                default:
+                    return ("draw");
+            }
+        default:
+            return ("The choice is invalid. Please write valid input");
```
Comment (roopaljasnani) [inline script.js:38]: You don't need to wrap the string in brackets in any of these return statements, directly do -
`return "...."`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock Paper Scissors Game</title>
+</head>
+<body>
+    <script src="script.js"></script>
+</body>
+</html>
```
Comment (roopaljasnani) [inline index.html:12]: new line at end of file

Comment (sahnavin123) [reply index.html:12]: I was wondering if you could kindly clarify the context for adding an extra blank line at the end of a file? I'm a bit confused about it.

Comment (roopaljasnani) [reply index.html]: Refer to my comment in the Recipes project

Comment (sahnavin123) [reply index.html]: added the changes

Comment (sahnavin123) [reply index.html]: Thank you, i have applied the changes

**Review by roopaljasnani — APPROVED** [review]
Code LGTM! 

Just observed however that the title of this PR does not convey what this PR is about, pls update it.

**Comment by sahnavin123** [comment]
> Code LGTM!
> 
> Just observed however that the title of this PR does not convey what this PR is about, pls update it.

Updated PR title. Thank you for the suggestion

**Review by devanuj97 — APPROVED** [review]

**Pull request merged**